server/controllers/Sessions.js

```diff
@@ -0,0 +1,78 @@
+import bcrypt from 'bcryptjs';
+import {
+  findUser,
+  // generateToken,
+  serverResponse,
+  serverError
+  // expiryDate
+} from '../helpers';
+// import models from '../database/models';
+
+// const { Session } = models;
+// const { findUser } = helpers;
+// const mobileDeviceIndicator = [
+//   'mobile',
+//   'andriod',
+//   'iphone',
+//   'tablet',
+//   'ipad',
+//   'ipod'
+// ];
+
+/**
+ *
+ *
+ * @class Sessions
+ */
+class Sessions {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @memberof User
+   * @returns {json}  object
+   */
+  static async create(req, res) {
+    try {
+      const { userLogin, password } = req.body;
+      if (!/\D/.test(userLogin)) {
+        return serverResponse(res, 400, {
+          message: 'user login must be a string'
+        });
+      }
+      const user = await findUser(userLogin);
```

Comment:
How about we start thinking of models as objects that can hold properties, which is what they are. It would be nice to have this method that query a User table on the associated model object. I need all `rambo-devs` to reason in this light.